scripts/test-files/http-server.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: http-server
+  namespace: kubevirt
+  labels:
+    app: http-server
+spec:
+  selector:
+    matchLabels:
+      app: http-server
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: http-server
+    spec:
+      volumes:
+        - name: nginx-config
+          configMap:
+            name: nginx-conf
+        - name: iso-dv
+          persistentVolumeClaim:
+            claimName: iso-dv
+      containers:
+      - name: http-server
+        image: nginxinc/nginx-unprivileged:latest
```

Comment:
Can we get the image from another registry than docker.io?

Reply:
This might lead to trouble in the future. On the other hand I don't really like publishing an image ourselves. Are there any other solutions?